package.json

```diff
@@ -40,12 +40,16 @@
   ],
   "main": "main.js",
   "scripts": {
-    "start": "webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
+    "start": "npm run build:templates && webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
     "test": "karma start karma.config.js",
-    "build": "webpack --mode production --config config/webpack.prod.mjs",
+    "build:templates": "gulp templates",
+    "watch:templates": "gulp watch",
+    "build": "npm run build:templates && webpack --mode production --config config/webpack.prod.mjs",
     "start:prod-local": "cd dist && http-server -p 4200 -c-1 -b",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
     "lint": "eslint .",
+    "e2e": "playwright test",
```

Comment:
Running this was failing for me until I ran `npx playwright install` (which seems to install browser binaries in the user's home-directory). Maybe we could make the process more straight forward by running `npx playwright install` automatically (either as a `postinstall` script or as a `pree2e` script).

Reply:
Added to prepare script:
`"prepare": "husky install && npx playwright install"`